internal/host/validator.go

```diff
@@ -128,8 +128,12 @@ func (c *validationContext) loadInventory() error {
 	if inventory == nil || err != nil {
 		return err
 	}
-	if inventory.CPU == nil || inventory.Memory == nil || len(inventory.Disks) == 0 {
-		return errors.New("Inventory is not valid")
+	if inventory.Memory == nil {
+		return errors.New("Inventory is not valid, Memory not detected")
+	}
+
+	if inventory.CPU == nil {
+		return errors.New("Inventory is not valid, CPU not detected")
 	}
```

Comment:
would it make sense to add a unit test for this change as well?

Reply:
Adding test for this is just adding an empty struct  to `inventory.Memory`  and `inventory.CPU`
we don't have any other / edge cases to this function. 

my guess seems not worth it.